rllib/utils/replay_buffers/tests/test_multi_agent_mixin_replay_buffer.py

```diff
@@ -187,19 +187,19 @@ def test_mixin_sampling_timesteps(self):
         buffer.add(batch)
         # Expect exactly 1 sample to be returned (the new batch).
         sample = buffer.sample(1)
-        assert type(sample) == MultiAgentBatch
+        assert isinstance(sample, MultiAgentBatch)
         self.assertTrue(len(sample) == 1)
         # Another replay -> Expect exactly 1 sample to be returned.
         sample = buffer.sample(1)
-        assert type(sample) == MultiAgentBatch
+        assert isinstance(sample, MultiAgentBatch)
         self.assertTrue(len(sample) == 1)
         # If we replay n times, expect roughly return batches of
         # len 1 (replay_ratio=1.0 -> 100% replayed samples -> 0 new and 1 old samples
         # on average in each returned value).
         results = []
         for _ in range(100):
             sample = buffer.sample(1)
-            assert type(sample) == MultiAgentBatch
```

Comment:
type() and isinstance are not always equivalent when there is parent and child classes?

Reply:
Thanks for your suggestions, I will fix that